governance/polls/Ratification Poll for Onboard-Amend Keeper Network - Tech-Ops (MIP63c4-SP1) - February 14, 2022.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for Onboard/Amend Keeper Network - Tech-Ops (MIP63c4-SP1) - February 14, 2022
+summary: This is an onboarding application for the tech-ops internal core unit.
```

Comment:
I'd probably rephrase this, something like: "This MIP onboards the Tech-Ops Core Unit (TECH-001) as part of the Maker Keeper Network."